python/tank/template.py

```diff
@@ -507,8 +565,8 @@ def read_templates(pipeline_configuration):
     :param pipeline_configuration: pipeline config object
 
     :returns: Dictionary of form {template name: template object}
-    """
-    
+    """    
+    multi_os_data_roots = pipeline_configuration.get_all_platform_data_roots()
```

Comment:
minor - variable name should be consistent - would probably prefer platform_data_roots